source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -267,11 +267,7 @@ To search for geospatial data within a specified shape use the ``geoWithin()``
 static utility method of the ``Filters`` builder class. The ``geoWithin()``
 method constructs a query with the ``$geoWithin`` query operator. 
 
-.. figure:: /includes/figures/geo_geometry.png
-   :alt: Area of Long Island we are searching for movie theaters
-
-The following example searches for movie theaters in the section of Long Island
-described in the figure above.
+The following example searches for movie theaters in the section of Long Island.
```

Comment:
```suggestion
The following example searches for movie theaters in a section of Long Island.
```

Issue: As the image has been removed, "the section of long island" doesn't have a reference.